Sources/Sentry/SentryOptions.m

```diff
@@ -39,35 +39,33 @@ @implementation SentryOptions {
     BOOL _enableTracingManual;
 }
 
-+ (NSArray<NSString *> *)defaultIntegrations
+NSMutableArray<NSString *> *
+sentry_defaultIntegrations(void)
```

Comment:
l: Why was this changed?
Just curious.

Reply:
I changed it to return a mutable array so that I can conditionally add the user feedback integration before installing all integrations, if the options property to configure its options is set (which implicitly enables the feature)